lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -207,7 +201,7 @@ class DetailsRenderer {
       // The value's type overrides the heading's for this column.
       switch (value.type) {
         case 'code': {
-          return this._renderCode(value);
+          return this._renderCode(value.value);
```

Comment:
🤔 not that it's new, but seems like we need a new local name for `value`

Reply:
> but seems like we need a new local name for `value`

haha, yeah. I actually like the local `value` and would rather change the property (e.g. `value.code`), but that's probably way too much for now :)